cli/src/main/java/org/apache/iotdb/tool/ImportCsv.java

```diff
@@ -419,115 +414,118 @@ private static void writeDataAlignedByTime(
-                                    type = headerTypeMap.get(headerNameMap.get(measurement));
-                                    if (type != null) {
-                                      Object valueTransed = typeTrans(value, type);
-                                      if (valueTransed == null) {
-                                        isFail.set(true);
-                                        System.out.println(
-                                            String.format(
-                                                "Line '%s', column '%s': '%s' can't convert to '%s'",
-                                                (records.indexOf(record) + 1),
-                                                measurement,
-                                                value,
-                                                type));
-                                      } else {
-                                        values.add(valueTransed);
-                                        measurements.add(headerNameMap.get(measurement));
-                                        types.add(type);
-                                      }
-                                    }
-                                  }
-                                });
-                        if (isFail.get()) {
-                          failedRecords.add(record.stream().collect(Collectors.toList()));
-                        }
-                        if (!measurements.isEmpty()) {
-                          try {
-                            if (timeFormatter == null) {
-                              try {
-                                times.add(Long.valueOf(record.get(timeColumn)));
-                              } catch (Exception e) {
-                                System.out.println(
-                                    "Meet error when insert csv because the format of time is not supported");
-                                System.exit(0);
-                              }
-                            } else {
-                              times.add(timeFormatter.parse(record.get(timeColumn)).getTime());
-                            }
-                          } catch (ParseException e) {
-                            e.printStackTrace();
-                          }
-                          typesList.add(types);
-                          valuesList.add(values);
-                          measurementsList.add(measurements);
-                          recordSize.getAndIncrement();
-                          if (recordSize.get() >= BATCH_SIZE) {
-                            writeAndEmptyDataSet(
-                                device, times, typesList, valuesList, measurementsList, 3);
-                            recordSize.set(0);
-                          }
-                        }
-                      });
-              writeAndEmptyDataSet(device, times, typesList, valuesList, measurementsList, 3);
-            });
+    final int BATCH_SIZE = 10000;
```

Comment:
We can put this constant to class static level

Reply:
ok, got this.